index.d.ts

```diff
@@ -655,7 +655,7 @@ export interface DefaultMetricsCollectorConfiguration {
  */
 export function collectDefaultMetrics(
 	config?: DefaultMetricsCollectorConfiguration
-): number;
+): NodeJS.Timeout;
```

Comment:
Do we not the the references comment at the top?

Reply:
Looks like it is not needed?

Reply:
Looks like it is not needed? Actually, it is possible to use something like `ReturnType` to not even specify it explicitly if you want that.

Reply:
`ReturnType<typeof global.setTimeout`? Might work, although since this package only works in Node it should be fine

Reply:
If the references part is not needed, great!

Reply:
Refactored to that